api/egl.xml

```diff
@@ -3559,5 +3559,10 @@
                 <enum name="EGL_PLATFORM_SCREEN_QNX"/>
             </require>
         </extension>
+        <extension name="EGL_QNX_image_native_buffer" supported="egl">
+            <require>
+                <enum name="EGL_NATIVE_BUFFER_QNX"/>
```

Comment:
Same issue here as the other extension - this just expresses a dependency on the enum, but there needs to be an actual definition of it with value 0x3551 in the corresponding \<enums> block above. Then I can generate the header update from egl.xml properly.

Reply:
I see now. thanks. Then do I need to remove changes to eglext.h?

Reply:
Preferably. I regenerate the headers when XML changes are merged - if people do it in their own branches it tends to lead to merge conflicts if several are in flight, as is happening right now.